blockchain/vm/contracts.go

```diff
@@ -98,6 +99,59 @@ var PrecompiledContractsIstanbul = map[common.Address]PrecompiledContract{
 	common.BytesToAddress([]byte{3, 255}): &validateSender{},
 }
 
+// PrecompiledContractsKore contains the default set of pre-compiled Klaytn
+// contracts based on Ethereum Berlin.
+var PrecompiledContractsKore = map[common.Address]PrecompiledContract{
+	common.BytesToAddress([]byte{1}):      &ecrecover{},
+	common.BytesToAddress([]byte{2}):      &sha256hash{},
+	common.BytesToAddress([]byte{3}):      &ripemd160hash{},
+	common.BytesToAddress([]byte{4}):      &dataCopy{},
+	common.BytesToAddress([]byte{5}):      &bigModExp{eip2565: true},
+	common.BytesToAddress([]byte{6}):      &bn256AddIstanbul{},
+	common.BytesToAddress([]byte{7}):      &bn256ScalarMulIstanbul{},
+	common.BytesToAddress([]byte{8}):      &bn256PairingIstanbul{},
+	common.BytesToAddress([]byte{9}):      &blake2F{},
+	common.BytesToAddress([]byte{3, 253}): &vmLog{},
+	common.BytesToAddress([]byte{3, 254}): &feePayer{},
+	common.BytesToAddress([]byte{3, 255}): &validateSender{},
+}
+
+var (
+	PrecompiledAddressesKore                []common.Address
+	PrecompiledAddressesIstanbulCompatible  []common.Address
+	PrecompiledAddressesByzantiumCompatible []common.Address
+)
+
+func init() {
+	for k := range PrecompiledContractsByzantiumCompatible {
+		PrecompiledAddressesByzantiumCompatible = append(PrecompiledAddressesByzantiumCompatible, k)
+	}
+
+	// For istanbulCompatible, need to support for vmversion0 contracts, too. VmVersion0 contracts are deployed before istanbulCompatible.
+	// they use byzantiumCompatible precompiled contracts because they were deployed before istanbulCompatible.
+	for k := range PrecompiledContractsIstanbulCompatible {
+		PrecompiledAddressesIstanbulCompatible = append(PrecompiledAddressesIstanbulCompatible, k)
+	}
+	PrecompiledAddressesIstanbulCompatible = append(PrecompiledAddressesIstanbulCompatible,
+		[]common.Address{common.BytesToAddress([]byte{10}), common.BytesToAddress([]byte{11})}...)
+
+	for k := range PrecompiledContractsKore {
```

Comment:
Why the byzantiumCompatible precompiled contracts are not added for this case?

Reply:
thanks, those addresses should be added at Kore too. updated at https://github.com/klaytn/klaytn/pull/1705/commits/3b127e0b64796534def79c0d55a15323aa8da6eb